docs/ops/queue.toml

```diff
@@ -11,7 +11,7 @@ prototype = "Q.N"
 prototype_set = "Q.N x"
 short = "The queue length"
 description = """
-Gets/sets the length of the queue.
+Gets/sets the length of the queue. The length is 1-indexed.
```

Comment:
not sure what this means? length is just the total number of elements in the queue, index doesn’t apply here, no?

Reply:
Yeah, I guess you're right. Was just trying to make a note to set it apart from the 0-indexed nature of the rest of them...